doc/ref/toc-ysh.md

```diff
@@ -237,7 +237,7 @@ X [Builtin Sub]   _buffer
 
 ```chapter-links-type-method
   [Primitive] Bool   Int   Float   Str   Slice   Range   
-  [Str]       X find(eggex)   X replace(eggex, template)
+  [Str]       X find(eggex)   replace()
```

Comment:
The `find(eggex)` breaks the link generation on `replace`. This is an issue in the `doctools/help_gen.py` `TOPIC_RE`. Fixing that is probably for another PR.

Reply:
Yeah I can take a look at this

I noted on Zulip that I want to fix a few other things about how these are formatted

https://oilshell.zulipchat.com/#narrow/stream/202008-oil-documentation/topic/Re-format.20doc.2Fref.20TOC.20.20a.20bit